lib/linters/decimal_zero.js

```diff
@@ -9,58 +9,67 @@ module.exports = {
 
     lint: function decimalZeroLinter (config, node) {
         var number;
-        var value;
-        var output = {
-            inclusion: 'with',
-            type: null
-        };
+        var result = [];
+        var self = this;
 
-        node.forEach('value', function (element) {
-            value = element.first('dimension');
-            value = value && value.first('number');
-        });
+        node.traverseByType('number', function (element) {
```

Comment:
Wrapped linting logic with `traverseByType()` to run linting on *each* number found in declaration.